src/node/setupServer.ts

```diff
@@ -1,49 +1,9 @@
 import { ClientRequestInterceptor } from '@mswjs/interceptors/ClientRequest'
 import { XMLHttpRequestInterceptor } from '@mswjs/interceptors/XMLHttpRequest'
 import { FetchInterceptor } from '@mswjs/interceptors/fetch'
-import { defaultMaxListeners, setMaxListeners } from 'node:events'
 import { RequestHandler } from '~/core/handlers/RequestHandler'
-import { SetupServerApi as BaseSetupServerApi } from './SetupServerApi'
+import { SetupServerApi } from './SetupServerApi'
 import { SetupServer } from './glossary'
-import { isNodeExceptionLike } from './utils/isNodeExceptionLike'
-
-class SetupServerApi extends BaseSetupServerApi implements SetupServer {
```

Comment:
we no longer need this hack, because we aren't exposing `O(handlers * c)` request clones

Reply:
I am very much happy to get rid of this.